src/pages/change-request-details/change-request-details.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { AxiosResponse } from 'axios';
+import { ChangeRequest, exampleStandardChangeRequest } from 'utils';
+import { apiFetch } from '../../shared/axios';
+import ChangeRequestDetailsView from '../../components/change-request-details/change-request-details';
+import './change-request-details.module.css';
+
+const ChangeRequestDetails: React.FC = () => {
+  interface ParamTypes {
+    id: string;
+  }
+  const { id } = useParams<ParamTypes>();
+  const [changeRequest, setChangeRequest] = useState<ChangeRequest>(exampleStandardChangeRequest); // store projects data
+
+  // Transforms given project data and sets local state
+  const updateData: (response: AxiosResponse) => void = (res) => {
+    setChangeRequest({
+      ...res.data,
+      dateSubmitted: new Date(res.data.dateSubmitted),
+      dateReviewed: new Date(res.data.dateReviewed),
+      dateImplemented: new Date(res.data.dateImplemented)
```

Comment:
Why do the dates need to be updated here?

Reply:
They were returning as strings because type conversion doesn't work as well as I'd like.

Reply:
Ahh I see I see